src/components/Popover/Popover.md

```diff
@@ -0,0 +1,82 @@
+### Examples
+```jsx
+import Popover from 'aws-northstar/components/Popover';
+import Stack from 'aws-northstar/layouts/Stack';
+import StatusIndicator from '../StatusIndicator';
```

Comment:
Please use absolute import aws-northstar/components/StatusIndicator so that consumers can just copy and paste into their code.